solutions/src/main/java/ru/sberbank/school/task03/Main.java

```diff
@@ -0,0 +1,42 @@
+package ru.sberbank.school.task03;
+
+import ru.sberbank.school.task02.*;
+import ru.sberbank.school.task02.exception.FxConversionException;
+import ru.sberbank.school.task02.util.*;
+
+import java.math.BigDecimal;
+
+import static ru.sberbank.school.task02.util.Symbol.USD_RUB;
+
+public class Main {
+
+    public static void main(String[] args) {
+
+        if (args.length != 3) throw new FxConversionException("Неверное число аргументов");
```

Comment:
&#39;if&#39; construct must use &#39;{}&#39;s.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a12569733>*